src/reducers/reducers.test.js

```diff
@@ -1,6 +1,6 @@
-import node_parameters from '../utils/data/node_parameters.mock.json';
-import node_task from '../utils/data/node_task.mock.json';
-import node_data from '../utils/data/node_data.mock.json';
+import nodeParameters from '../utils/data/node_parameters.mock.json';
```

Comment:
any particular reason as to why we are changing the naming here? I think maintaining snake case to follow the name of the file would allow this variable to be more untuitive to imitate a data response.

Reply:
@studioswong  - @bru5 asked me to make it camel case. @bru5 can you comment why and if it's ok to revert?

Reply:
Yeah I raised this since the snake case naming style here seems to go against the standards followed in the rest of the project and it wasn't clear as to why, so I thought it had been by mistake.

We have recently written the project standards down in the [style guide](https://github.com/quantumblacklabs/kedro-viz/blob/main/STYLE_GUIDE.md) and if there's a good reason to we can update it, but we should really try avoid changing things too often or adding special cases as it defeats the purpose.

Anyway since this is already snake case in main, we should probably address any reasoning behind this approach later then.